eventbridge-pipes-amazonmq-to-eventbridge-sam/eventbridge-pipes-amazonmq-to-eventbridge-sam-pattern.json

```diff
@@ -0,0 +1,60 @@
+{
+  "title": "Amazon MQ broker as a producer for a Amazon EventBridge Message Bus using Amazon EventBridge Pipes",
+  "description": "This pattern showcases the use of Amazon EventBridge Pipes to forward events to an Amazon EventBridge custom Message Bus. Using this pattern, events placed on a queue on an Amazon MQ (ActiveMQ) broker are consumed by Amazon EventBridge Pipes which performs a basic transformation and places events on a Amazon EventBridge custom Message Bus. A rule defined on the Amazon EventBridge custom Message Bus writes events received to an Amazon CloudWatch Log Group.",
```

Comment:
Description also exceeds the maximum length of 150 characters

Reply:
Adjusted property `description` to 126 characters in commit https://github.com/aws-samples/serverless-patterns/commit/9175bc0dbb7bf99a2290fc4e9246968277135cc1